schema/crdb/3.0.0/up.sql

```diff
@@ -20,17 +12,36 @@ SELECT CAST(
 );
 
 ALTER TABLE omicron.public.ip_pool
+    ADD COLUMN IF NOT EXISTS is_default BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
Yes. I think I mentioned it at control plane huddle but I noted it was a bad idea. There isn't really a reason to do it this way, I can change it to use 3 new migrations.

Reply:
Changed to 3.0.1-3.0.3 instead of modifying 3.0.0 in [e3c56c4](https://github.com/oxidecomputer/omicron/pull/3985/commits/e3c56c45faabe21076da60488f283aeef2b9a5c5)